src/inpage-panel.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <title>Firefox Relay</title>
+    <link rel="stylesheet" href="/css/relay.css" />
+    <link rel="stylesheet" href = "/css/first-run.css" />
```

Comment:
```suggestion
```

Reply:
Removed in ed515009a70c4fd3b452eff9955d37539efb41bc